yt_dlp/extractor/servus.py

```diff
@@ -63,86 +62,51 @@ class ServusIE(InfoExtractor):
+            countries = ', '.join(attrs.get('blockedCountries', ['Unknown']))
+            raise ExtractorError(
+                'Video is geo restricted (restricted countries: %s). '
+                'Try bypassing with --geo-bypass-country option, VPN or --proxy option' % countries, expected=True)
 
         formats = []
-        thumbnail = None
-        for resource in video['resources']:
-            if not isinstance(resource, dict):
-                continue
-            format_url = url_or_none(resource.get('url'))
-            if not format_url:
-                continue
-            extension = resource.get('extension')
-            type_ = resource.get('type')
-            if extension == 'jpg' or type_ == 'reference_keyframe':
-                thumbnail = format_url
-                continue
+        format_url = url_or_none(attrs.get('videoUrl'))
+        if format_url:
             ext = determine_ext(format_url)
-            if type_ == 'dash' or ext == 'mpd':
+            if ext == 'mpd':
                 formats.extend(self._extract_mpd_formats(
                     format_url, video_id, mpd_id='dash', fatal=False))
-            elif type_ == 'hls' or ext == 'm3u8':
+            elif ext == 'm3u8':
                 formats.extend(self._extract_m3u8_formats(
-                    format_url, video_id, 'mp4', entry_protocol='m3u8_native',
-                    m3u8_id='hls', fatal=False))
-            elif extension == 'mp4' or ext == 'mp4':
+                    format_url, video_id, 'mp4', m3u8_id='hls', fatal=False))
+            elif ext == 'mp4':
                 formats.append({
                     'url': format_url,
-                    'format_id': type_,
-                    'width': int_or_none(resource.get('width')),
-                    'height': int_or_none(resource.get('height')),
                 })
         self._sort_formats(formats)
 
-        attrs = {}
-        for attribute in video['attributes']:
-            if not isinstance(attribute, dict):
-                continue
-            key = attribute.get('fieldKey')
-            value = attribute.get('fieldValue')
-            if not key or not value:
-                continue
-            attrs[key] = value
-
-        title = attrs.get('title_stv') or video_id
-        alt_title = attrs.get('title')
-        description = attrs.get('long_description') or attrs.get('short_description')
+        title = attrs.get('title') or video_id
+        description = attrs.get('description')
         series = attrs.get('label')
         season = attrs.get('season')
         episode = attrs.get('chapter')
-        duration = float_or_none(attrs.get('duration'), scale=1000)
-        season_number = int_or_none(self._search_regex(
-            r'Season (\d+)', season or '', 'season number', default=None))
-        episode_number = int_or_none(self._search_regex(
-            r'Episode (\d+)', episode or '', 'episode number', default=None))
```

Comment:
Do you know why these were removed? These regexes seem to still work

Reply:
dont just mark review as resolved without addressing them